server/controllers/admin/userManagementController.test.ts

```diff
@@ -25,19 +27,53 @@ describe('UserManagementController', () => {
   })
 
   describe('index', () => {
+    const hrefPrefix = `${paths.admin.userManagement.index({})}?`
+
     it('renders the index page with all the users', async () => {
       const users = userFactory.buildList(1)
 
-      userService.getUsers.mockResolvedValue(users)
+      const paginatedResponse = paginatedResponseFactory.build({
+        data: users,
+      }) as PaginatedResponse<ApprovedPremisesUser>
```

Comment:
It's a bit annoying having to cast this but I've had a go and I can't work out a way around it